oag/src/main/java/org/owasp/oag/config/FileConfigLoader.java

```diff
@@ -69,4 +160,11 @@ protected MainConfig load(InputStream defaultSettingsStream, InputStream userCon
 
         return finalConfig;
     }
+
+    /**
+     * Enables support to load configuration file via unsafe https connections.
+     */
+    public void enableUnsafeHttps() {
```

Comment:
Is this really unsafe?
Given you have a proper Truststore installed, I'd say this is as safe as filesystem. SO I would rename to enableHttps...